src/Voron/Data/Tables/Table.cs

```diff
@@ -1865,7 +1868,7 @@ void IEnumerator.Reset()
             // in IO deprived setups.
             // https://issues.hibernatingrhinos.com/issue/RavenDB-21106
 
-            // If we page fault on every page (specially in cold scenarios) we would be wasting a lot. Therefore we should
+            // If we page fault on every page (specially in cold scenarios) we would be wasting a lot. Therefore, we should
             // iterate with prefetching enabled.
             using var it = fst.Iterate(prefetch: true);
```

Comment:
I see the above comment but just to be absolutely sure - we do want to have `prefetch: true` explicitly defined here, right? (regardless `_prefetch`)

Reply:
Nice catch. No we don't I missed that one.

Reply:
OK. Given the comment it made sense to set it to `true` but wanted to double check with you